### PR TITLE
Bug 2038088: Remove references to ipa-downloader

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

**Comment by dtantsur** [comment]
/lgtm

Will need a valid bug to get merged in the near future.

File: provisioning/baremetal_pod_test.go [hunk 1]
```diff
@@ -99,16 +98,16 @@ func TestNewMetal3InitContainers(t *testing.T) {
...
-					Name:  "metal3-ipa-downloader",
-					Image: images.IpaDownloader,
+					Name:  "metal3-static-ip-set",
+					Image: images.StaticIpManager,
 				},
 				{
-					Name:  "metal3-machine-os-downloader",
-					Image: images.MachineOsDownloader,
+					Name:  "machine-os-images",
+					Image: images.IronicAgent,
```
Comment (zaneb) [inline provisioning/baremetal_pod_test.go:106]: This doesn't look right - machine-os-images does not use the IronicAgent image, it uses the MachineOSImages image.

It appears the test is only checking the number of containers... I think we should test the contents or just put the number here.

Comment (elfosardo) [reply provisioning/baremetal_pod_test.go:106]: I actually test the names of the containers, although I didn't add the test here, so the names are correct
I probably overlooked the Image name by mistake, it was not intentional
I'm going to fix this and maybe we can add the check on the containers names in a different PR

**Comment by zaneb** [comment]
Thanks for handling this!
/approve

**Pull request merged**